AzureMonitorAgent/agent.py

```diff
@@ -266,7 +266,15 @@ def install():
     exit_code, output = run_command_with_retries_output(OneAgentInstallCommand, retries = 15,
                                          retry_check = retry_if_dpkg_locked,
                                          final_check = final_check_if_dpkg_locked)    
-    
+    # set task limits in suse
+    vm_dist, vm_ver = find_vm_distro('Install')
+    if vm_dist.lower().startswith('suse'):
+        run_command_and_log("mkdir -p /etc/systemd/system/mdsd.service.d")
+        run_command_and_log("touch /etc/systemd/system/mdsd.service.d/override.conf")
+        run_command_and_log("echo '[Service]' > /etc/systemd/system/mdsd.service.d/override.conf")
+        run_command_and_log("echo 'TasksMax=10000' >> /etc/systemd/system/mdsd.service.d/override.conf")
+        run_command_and_log("systemctl daemon-reload")
```

Comment:
could we check the exit code(s)? if this fails, sles12 agent will be broken anyways; might as well fail fast and let customer know rather than proceeding